src/protobuf/rpc.proto

```diff
@@ -15,3 +19,42 @@ enum CheckVersionStatus {
 message CheckVersionRequest { string version_str = 1; }
 
 message CheckVersionReply { CheckVersionStatus status = 1; }
+
+/// Define the information required for CreateTable !!!
+
+enum CreateTableStatus {
+  SUCCESS = 0;
+  FAIL = 1;
+}
+
+enum FieldType {
+  // value type
+  INT = 0;
+  BIGINT = 1;
+  FLOAT = 2;
+  DOUBLE = 3;
+  // string type
+  STRING = 4;
+  // datatime type
+  TIMESTAMP = 5;
+  DATE = 6;
+  TIME = 7;
+  DATATIME = 8;
+}
+
+message Field {
+  string name = 1;
+  FieldType type = 2;
+}
+
+message TableSchema {
+  string table_name = 1;
+  repeated Field fields = 2;
+}
+
+message CreateTableRequest { TableSchema table_schema = 1; }
```

Comment:
It seems that we needn't TableSchema right? We can use CreateTableRequest directly:
```proto
message CreateTableRequest {
  string table_name = 1;
  repeated Field fields = 2;
}
```